docs/changelog.md

```diff
@@ -894,7 +894,7 @@ description: |
 
 - Development
   - Extracted parser and syntax tree into `typst-syntax` crate
-  - The `World::today` implementation of Typst dependants may need fixing if
+  - The `World::today` implementation of Typst dependents may need fixing if
```

Comment:
I'm aware that some projects don't like changing changelogs. Happy to drop this...